### PR TITLE
Refs[Task][35715] Làm modal trang home

### PR DESCRIPTION
**Pull request by duyvu0302**

### CODE REVIEW

**Comment by duyvu0302** [comment]
Ready

File: src/pug/pages/index.pug [hunk 1]
```diff
@@ -1,7 +1,22 @@
 extend ../base/base.pug
+include ../components/modal.pug
 include ../components/carousels.pug
+
+//- mixin titleSection(title,attribute)
+//-     .titleSection.py-5.d-flex.justify-content-between
+//-         div.d-flex
```
Comment (manhnt-0960) [inline src/pug/pages/index.pug]: k dùng xóa luôn e

**Review by manhnt-0960 — COMMENTED** [review]
Refs [task] [id redmine]

**Comment by duyvu0302** [comment]
ready

**Comment by duyvu0302** [comment]
ready

**Pull request merged**